### PR TITLE
Added filequery as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

File: recipes/filequery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - pip
+  run:
+    - python >=3.8
+    - python-duckdb ==0.9.1
+    - importlib-metadata ==6.8.0
+    - linkify-it-py >=2.0.2
+    - markdown-it-py ==3.0.0
+    - mdit-py-plugins ==0.4.0
+    - mdurl >=0.1.2
+    # numpy 1.26.1 not available on Conda Forge, loosening pin here
```
Comment (ocefpaf) [inline recipes/filequery/meta.yaml:33]: I see upstream pins all dependencies down to the micro version :-(

Comment (thewchan) [reply recipes/filequery/meta.yaml:33]: I hate it when they do that, but some folks are very paranoid and maybe they are justified; python packaging being what they are

Comment (ocefpaf) [reply recipes/filequery/meta.yaml:33]: It is not only packaging. The ecosystem is "more accessible" nowadays, one can Google, find "a repo," and report "problems" to anyone. The quotes there is b/c very few actually take the time to: find the right repo, study the problem and make a useful report and, more important than everything else, be polite with the fellow developer.

Sorry for the rant but people to try to defend from this by pinning will ended up falling in the same trap and getting the same bad error reports later on...

**Pull request merged**